db/connection.js

```diff
@@ -15,14 +15,29 @@ connection.connect(function(err) {
   } console.log ('Connected to MYSQL DB');
 });
 
-const mockAddress =  `${faker.address.streetAddress()}, ${faker.address.city()}, ${faker.address.country()}`;
-const queryStr = `INSERT INTO map (address, relative_location, phone_number, business_id) VALUES('${mockAddress}', '${mockAddress}', 12345, 2)`;
+
+// const mockAddress =  `${faker.address.streetAddress()}, ${faker.address.city()}, ${faker.address.country()}`;
+// const queryStr = `INSERT INTO map (address, relative_location, phone_number, business_id) VALUES('${mockAddress}', '${mockAddress}', 12345, 2)`;
+// connection.query(queryStr, (err, results) => {
+//   if (err) {
+//     throw err;
+//   } else {
+//     console.log('INSERT FUNCTION SUCCESS', results);
+//   }
+// });
+
```

Comment:
Thu, everything looks great. Only reason I didn't accept your PR is I was hoping you could remove the cruft above.